03_launch_mgmt_cluster.sh

```diff
@@ -39,14 +36,31 @@ source lib/ironic_basic_auth.sh
 # BMO and Ironic deployment functions
 # ------------------------------------
 
+function deployIronicAndOrBMO() {
```

Comment:
```suggestion
deploy_ironic_and_or_bmo() {
```
No need to add `function`, and camel case -> snake case.